extensions/linz/schema.json

```diff
@@ -20,7 +20,14 @@
   "definitions": {
     "linz": {
       "type": "object",
-      "required": ["title", "linz:security_classification", "linz:created", "linz:lifecycle", "linz:updated"],
+      "required": [
+        "title",
+        "linz:security_classification",
+        "linz:created",
+        "linz:lifecycle",
+        "linz:providers",
+        "linz:updated"
```

Comment:
Since we have such a long list it might make sense to order it alphabetically. What do you think?